tests/unit/provisioner/test_dockerprovisioner.py

```diff
@@ -31,10 +32,22 @@
 
 logging.getLogger("sh").setLevel(logging.WARNING)
 
-pytestmark = pytest.mark.skipif(
-    distutils.version.LooseVersion(ansible.__version__) <
-    distutils.version.LooseVersion('2.0'),
-    reason='No Ansible v2 executable found - skipping docker tests')
+
+def ansible_v1():
+    d = distutils.version
+    return (d.LooseVersion(ansible.__version__) < d.LooseVersion('2.0'))
+
+
+def get_docker_executable():
+    return distutils.spawn.find_executable('docker') is None
```

Comment:
Maybe nitpicking, but I would have done the opposite: if there is no docker executable returns False, otherwise True.